.github/workflows/rebuild_native_libs.yml

```diff
@@ -17,6 +17,26 @@ on:
 # and pushes them to the current branch.
 
 jobs:
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build Rust on Windows
+        run: |
+          cd core/rust/qdbr
+          cargo build --release
+      - name: log git status
+        run: |
+          git status
+      - name: Copy Windows Rust Library to the final directory
+        run: |
+          cp core/rust/qdbr/target/release/questdbr.dll core/src/main/resources/io/questdb/bin/windows/libquestdbr.dll
```

Comment:
Windows lib names should not start with lib.
This breaks convention and is quite confusing.
Not a bug, but breaks the principle of least surprise.

Reply:
Ah, ok. I've noticed this is because the CMake lib names are wrong on Windows.

Probably worth fixing those and having everything consistent and following platform naming standards too as well, without `lib` prefix on Windows.

Reply:
You should be able to do this by setting:
```
if(WIN32)
    set(CMAKE_SHARED_LIBRARY_PREFIX "")
endif()
```

Towards the top of the `CMakeLists.txt` file.

Reply:
i am going to ingnore this until this becomes an issue later, we pulled thru for 10 years with these library names :) we can pull some more :)